python/fbprophet/forecaster.py

```diff
@@ -892,6 +888,10 @@ def plot(self, fcst, ax=None, uncertainty=True, plot_cap=True, xlabel='ds',
         -------
         A matplotlib figure.
         """
+        from matplotlib import pyplot as plt
+        from matplotlib.dates import MonthLocator, num2date
+        from matplotlib.ticker import FuncFormatter
+
```

Comment:
We can generalize it with one single function. This suppose to work with both py2 and py3
```python
def import_matplotlib():
    from matplotlib import pyplot as plt
    from matplotlib.dates import MonthLocator, num2date
    from matplotlib.ticker import FuncFormatter
    global plt, MonthLocator, num2date, FuncFormatter # escape local function scope

```